ui/slider.js

```diff
@@ -558,7 +558,7 @@ return $.widget( "ui.slider", $.ui.mouse, {
 		var max = this.options.max,
 			min = this._valueMin(),
 			step = this.options.step,
-			aboveMin = Math.floor( ( max - min ) / step ) * step;
+				aboveMin = Math.floor( ( +( max - min ).toFixed(5) ) / step ) * step;
```

Comment:
Let's not use arbitrary precision. See https://github.com/jquery/jquery-ui/blob/18e301f4e29c2080e9aa9dac87c00dee137cb6c9/ui/spinner.js#L328.